tune_sklearn/tune_basesearch.py

```diff
@@ -307,22 +307,33 @@ def _check_is_fitted(self, method_name, check_refit=True):
         Args:
             method_name (str): String of the method name called from the user.
 
-            check_refit(bool)=True: Whether to also check for `self.refit`
-            param.
+            check_refit(bool/str)=True: Whether to also check for `self.refit`
```

Comment:
hmm i think this doesn't render properly on the documentation
```suggestion
            check_refit (bool/str): Whether to also check for `self.refit`
```

Reply:
Thanks, changed